src/store/modules/stores.js

```diff
@@ -3,6 +3,44 @@ import stores from '@/services/api/stores'
 import { indexById, onlyHandleAPIError } from '@/store/helpers'
 import router from '@/router'
 
+const statusList = {
```

Comment:
This does not belong in state (it never changes...), it's a UI consideration.

**Solution**: if it's used in multiple places, just pop it in a `.js` file somewhere with `export default { ...` and load it where it is used

Reply:
Yeah, probably it's better to have this in a file.. but where? Just besides a component could be fine?

I'd also be fine if we import this file in the store and make it available via getter. I did something similar with the locales, it imports `i18n` and makes the locales available via getter. Makes it possible to mix in more properties.

Reply:
Ah yeah, I didn't really like the locales approach either :p (as it returns in the format that the autocomplete component wants them in which is UI-specific).

My general view would be that data that does not change does not belong in state. Having to connect to vuex for static data is extra unneeded complication.

Reply:
Would be happy about a PR to improve the i18n static data handling!